src/transformers/models/wav2vec2/feature_extraction_wav2vec2.py

```diff
@@ -33,7 +33,7 @@ class Wav2Vec2FeatureExtractor(SequenceFeatureExtractor):
     r"""
     Constructs a Wav2Vec2 feature extractor.
 
-    This feature extractor inherits from :class:`~transformers.Wav2Vec2FeatureExtractor` which contains most of the
+    This feature extractor inherits from :class:`~transformers.SequenceFeatureExtractor` which contains most of the
```

Comment:
```suggestion
    This feature extractor inherits from :class:`~transformers.feature_extraction_sequence_utils.SequenceFeatureExtractor` which contains most of the
```

Reply:
Since one cannot import `SequenceFeatureExtractor` directly, we need to import  from the file here. This way the docs are correctly linked.